packages/gatsby-theme-newrelic/src/utils/generateUUID.js

```diff
@@ -0,0 +1,10 @@
+const generateUUID = () => {
```

Comment:
what is the UUID?

Reply:
Since we don't have the concept of a logged in user, we need some other means of creating a stable user ID that persists between sessions. This function generates a [v4 UUID](https://en.wikipedia.org/wiki/Universally_unique_identifier#Version_4_(random)) that we can then use to assign as the user ID. This user ID is persisted in local storage so that we can maintain the same ID between page refreshes.

Does that help?

Reply:
yes!